tools/meson-toolchain.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
```

Comment:
MXE already includes a plugin that wraps mesa (see `plugins/mesa-wrapper`). It would probably be a better idea to move it to `src/` and use for these meson-powered projects.

Reply:
ps. I also added you to this repo, so you have all the rights. Allowing us to work together.

Reply:
Lets hope they merge your PR, so we can continue further with this GTK update. And I will ditch my script.

Reply:
This is now solved. We moved the plugin to the src folder. And indeed make a build dependency out of it.